content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -216,6 +216,10 @@ kubectl get nodes -o json | jq -c 'path(..)|[.[]|tostring]|join(".")'
 
 # Produce a period-delimited tree of all keys returned for pods, etc
 kubectl get pods -o json | jq -c 'path(..)|[.[]|tostring]|join(".")'
+
+# Produce ENV for all pods
+# Make sure to set your default namespace or this will run on system pods
```

Comment:
I would also comment that this relies on the Pod having a default container (from kubectl's perspective), and that the `env` utility is available inside that container.

Reply:
Updated.